projects/components/src/tabs/navigable/navigable-tab-group.component.ts

```diff
@@ -57,7 +57,7 @@ export class NavigableTabGroupComponent implements AfterContentInit {
   });
 
   public onTabClick(tab: NavigableTabComponent): void {
-    this.navigationService.navigateWithinApp([tab.path], this.activatedRoute);
+    this.navigationService.navigateWithinApp([tab.path], this.activatedRoute, [], tab.replaceHistory);
```

Comment:
did i accidentally forget to remove it? This function shouldn't be used. We need to add this option to the `buildNavigationParams`

Reply:
yep, this is nested inside an ht-link and should be deleted here and in the template

Reply:
@anandtiwary fixed this. Used ht-link in the component.